cgmanifest.json

```diff
@@ -0,0 +1,1634 @@
+            "Version": "0.33.0"
+          }
+        }
+      },
+      {
+        "Component": {
+          "Type": "NuGet",
+          "NuGet": {
+            "Name": "Xamarin.Android.Arch.Core.Common",
+            "Version": "1.0.0.1"
+          }
+        }
+      },
+      {
+        "Component": {
+          "Type": "NuGet",
+          "NuGet": {
+            "Name": "Xamarin.Android.Arch.Lifecycle.Common",
+            "Version": "1.0.3.1"
+          }
+        }
+      },
+      {
+        "Component": {
+          "Type": "NuGet",
+          "NuGet": {
+            "Name": "Xamarin.Android.Arch.Lifecycle.Runtime",
+            "Version": "1.0.3.1"
+          }
+        }
+      },
+      {
+        "Component": {
+          "Type": "NuGet",
+          "NuGet": {
+            "Name": "Xamarin.Android.Support.Animated.Vector.Drawable",
+            "Version": "27.0.2.1"
+          }
+        }
+      },
+      {
+        "Component": {
+          "Type": "NuGet",
+          "NuGet": {
+            "Name": "Xamarin.Android.Support.Annotations",
+            "Version": "27.0.2.1"
+          }
+        }
+      },
+      {
+        "Component": {
+          "Type": "NuGet",
+          "NuGet": {
+            "Name": "Xamarin.Android.Support.Compat",
+            "Version": "27.0.2.1"
+          }
+        }
+      },
+      {
+        "Component": {
+          "Type": "NuGet",
+          "NuGet": {
+            "Name": "Xamarin.Android.Support.Core.UI",
+            "Version": "27.0.2.1"
+          }
+        }
+      },
+      {
+        "Component": {
+          "Type": "NuGet",
+          "NuGet": {
+            "Name": "Xamarin.Android.Support.Core.Utils",
+            "Version": "27.0.2.1"
+          }
+        }
+      },
+      {
+        "Component": {
+          "Type": "NuGet",
+          "NuGet": {
+            "Name": "Xamarin.Android.Support.Design",
+            "Version": "27.0.2.1"
+          }
+        }
+      },
+      {
+        "Component": {
+          "Type": "NuGet",
+          "NuGet": {
+            "Name": "Xamarin.Android.Support.Fragment",
+            "Version": "27.0.2.1"
+          }
+        }
+      },
+      {
+        "Component": {
+          "Type": "NuGet",
+          "NuGet": {
+            "Name": "Xamarin.Android.Support.Media.Compat",
+            "Version": "27.0.2.1"
+          }
+        }
+      },
+      {
+        "Component": {
+          "Type": "NuGet",
+          "NuGet": {
+            "Name": "Xamarin.Android.Support.Transition",
+            "Version": "27.0.2.1"
+          }
+        }
+      },
+      {
+        "Component": {
+          "Type": "NuGet",
+          "NuGet": {
+            "Name": "Xamarin.Android.Support.v4",
+            "Version": "27.0.2.1"
+          }
+        }
+      },
+      {
+        "Component": {
+          "Type": "NuGet",
+          "NuGet": {
+            "Name": "Xamarin.Android.Support.v7.AppCompat",
+            "Version": "27.0.2.1"
```

Comment:
Any way to fix auto detection?
That will be most likely stale / out of sync pretty quick if it's manual.

Reply:
That should be auto-detected by CG with the way it is implemented on CI now. Resolving